setup.cfg

```diff
@@ -0,0 +1,31 @@
+[metadata]
+name = tool
+version = attr: tool.version.__version__
```

Comment:
Just like Tomas sad in other PR, the setuptools_scm would be nice, but we can leave this for CI/testing/tooling discussion.

Reply:
I'd rather see such a change in different PR.